app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::Base
+  def check_user_authority_on_blog
+    unless current_user.can_access?(@blog)
+      flash[:notice] = 'Sorry you are not authorised to access this resource'
+      redirect_to blogs_path
+    end
+  end
```

Comment:
Great to see some authorization. 

Let's say a requirement was added so that users could allow certain other users to make post on their blogs. How would you solve this? (Just write some quick notes here on roughly how you'd approach it, no need to write the code.)

Reply:
Happy to know that.